dbsnp-importer/src/test/resources/application.properties

```diff
@@ -1,11 +1,10 @@
 spring.datasource.driver-class-name=org.hsqldb.jdbcDriver
-spring.datasource.url=jdbc:hsqldb:mem:db;DB_CLOSE_DELAY=-1
+spring.datasource.url=jdbc:hsqldb:mem:db;sql.syntax_pgs=true;DB_CLOSE_DELAY=-1
```

Comment:
is it syntactically required for this directive to be on the same line as the datasource URL (because i do see a similar directive that was deleted from the file further down)?

Reply:
Yes, it has to be in the URL, it doesn't work as a separate directive.